selectrum.el

```diff
@@ -268,6 +281,15 @@ If PREDICATE is non-nil, then it filters the collection as in
    (t
     (error "Unsupported collection type %S" (type-of collection)))))
 
+(defun selectrum--count-info ()
+  "Return a string of count information to be prepended to prompt."
+  (let ((total (length selectrum--refined-candidates))
+        (current (1+ (or selectrum--current-candidate-index -1))))
+    (cond ((eq selectrum-count-style 'matches)
+           (format "%d " total))
+          ((eq selectrum-count-style 'current/matches)
+           (format "%d/%d " current total))
+          (t ""))))
```

Comment:
Would it be possible to add some leading whitespace, like Ivy does, so that the user input area doesn't shift around horizontally as you type?

Reply:
Done. Ivy actually uses trailing whitespaces, and so did I. I found leading whitespaces looks bad.